src/test/resources/n.json

```diff
@@ -35,7 +35,7 @@
     "num2" : 1,
     "r" : 1.846153846153846E-4,
     "x" : 0.007690091988585015,
-    "r1" : 1.0526315789473686
+    "r1" : 1.0526315789473684
```

Comment:
why is it different?

Reply:
Because I have introduced `baseRatio` as an intermediate.